### DISCUSSION TITLE
Regarding the "Target Version" field on the patches list

### DISCUSSION THREAD

**Discussion by kairusds** [ReVanced/revanced-discussions#907] [General] [answered]
There should be an included minimum version besides the target version field since `all` does not really mean that the patch applies to all past versions.

**Comment by oSumAtrIX** [comment, marked as answer]
With all it is meant, you can patch the latest versions, sometimes even any

**Reply by kairusds** [reply]
Latest versions like the ones with the most noticeable UI changes?
